platforms/quorum/configuration/roles/create/generate_crypto_tessera/tasks/main.yaml

```diff
@@ -0,0 +1,49 @@
+# This task checks if the tm crypto material is already stored in the vault or not
+- name: Check if tm key is already present in the vault
+  shell: |
+    vault read -field=tm.key secret/{{ component_ns }}/crypto/{{ component_name }}/transaction/
```

Comment:
The path will change {{ component_name }} shall be replaced to {{ peer.name }} with a loop over peers. I am currently doing this change for RAFT.